bootstrap

```diff
@@ -202,18 +205,24 @@ while (true) {
 
   $ch = curl_init("http://$AWS_LAMBDA_RUNTIME_API/2018-06-01/runtime/invocation/$invocation_id/response");
 
+  $isALB = array_key_exists("elb", $event['requestContext']);
+  if ($isALB) { // Add Headers For ALB
+    $status = $response["statusCode"];
+    $response["statusDescription"] = "$status ". $http_codes[$status];
```

Comment:
What happens if the status code isn't in `http_codes`? I think PHP would throw an error.

Maybe only append the `http_codes` message if the status exists?

Reply:
Hey @txase We could put it inside a try statement

example:
```

try {
    $response["statusDescription"] = "$status ". $http_codes[$status];
} catch (Exception $e) {
    $response["statusDescription"] = "$status Unknown"
}
```
Something like that what do you think?

Reply:
What about this:

```php
if (array_key_exists($status, $http_codes)) {
    $response["statusDescription"] = "$status ". $http_codes[$status];
} else {
    $response["statusDescription"] = "$status Unknown";
}
```

I think it's better to check for conditions explicitly rather than capture all errors (including those you didn't mean to capture).

Reply:
Seems great to me :+1:

Reply:
Cool, can you update the PR and test it out? 🙏

Reply:
Updated and tested.  Everything worked great.
I Couldnt find any HTTP code to test the unknown status code, but in theory it should work withou an issue.